.github/workflows/ci.yml

```diff
@@ -97,6 +97,27 @@ jobs:
         env:
           BOOTSTRAPVERSION: ${{ matrix.bootstrap }}
 
+  test-docs:
+    name: Docs
+    runs-on: ubuntu-latest
+    needs:
+      - test
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 14
+      - name: Install Dependencies of Ember Bootstrap addon
+        uses: bahmutov/npm-install@v1
```

Comment:
This is needed because `ember-bootstrap` is not declared as a regular dependency on the docs app. Instead it uses a discovery mechanism provided by Ember CLI. At least if I understood correctly what those lines are doing: https://github.com/ember-bootstrap/ember-bootstrap/blob/56f723c9209b220751bc1ffdb58e7d7a44323582/docs/package.json#L80-L84

We may want to refactor away from that pattern and declare it as a regular dependency using `file:` or `link:` protocol.

Reply:
Agree. Actually converting to a real monorepo probably? Kinda inevitable when eventually refactoring to a v2 addon...

Reply:
Yeah. Let's reopen the discussion in #1304.